<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>volumeTool</class>
 <widget class="QWidget" name="volumeTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="content" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#content
{
	background:#ffffff;
	border-radius:5px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sloderBox" native="true">
      <widget class="QWidget" name="inSloder" native="true">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>25</y>
         <width>4</width>
         <height>180</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#ECECEC;</string>
       </property>
      </widget>
      <widget class="QWidget" name="outSloder" native="true">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>25</y>
         <width>4</width>
         <height>180</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#1ECC94;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sloderBt">
       <property name="geometry">
        <rect>
         <x>33</x>
         <y>18</y>
         <width>14</width>
         <height>14</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#1ECC94;
border-radius:7px;
color:#ffffff;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-bottom:1px solid silver;</string>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton:hover
{
	color:#1ECD98;
}</string>
      </property>
      <property name="text">
       <string>音量</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
